websockets/helpers.go

```diff
@@ -11,3 +15,51 @@ func must(rt *sobek.Runtime, err error) {
 		common.Throw(rt, err)
 	}
 }
+
+func isString(o *sobek.Object, rt *sobek.Runtime) bool {
+	return o.Prototype().Get("constructor") == rt.GlobalObject().Get("String")
+}
+
+func isArray(o *sobek.Object, rt *sobek.Runtime) bool {
+	return o.Prototype().Get("constructor") == rt.GlobalObject().Get("Array")
+}
+
+func isUint8Array(o *sobek.Object, rt *sobek.Runtime) bool {
+	return o.Prototype().Get("constructor") == rt.GlobalObject().Get("Uint8Array")
+}
+
+func isDataView(o *sobek.Object, rt *sobek.Runtime) bool {
+	return o.Prototype().Get("constructor") == rt.GlobalObject().Get("DataView")
+}
+
+func isBlob(o *sobek.Object, rt *sobek.Runtime) bool {
+	return o.Prototype().Get("constructor") == rt.GlobalObject().Get("Blob")
+}
```

Comment:
I am pretty sure this only works as in your tests you set Blob to be property of the global object. But that isn't how `import { Blob } from ...` works now or works with Native ESM.

I am not strictly against importing websockets setting `Blob` on the global object. Another alternative is to have the Blob "import" be saved on the module instance and use that to chec if it is the same.

Arguably at some future case where Blob is internal it will (like SetTimeout) be available globally.

Reply:
Nice catch! 🙇🏻  Let's go with:

> Another alternative is to have the Blob "import" be saved on the module instance and use that to check if it is the same.

As it is still experimental, I think it's better to follow the common procedure, and leave `Blob` only available with the import, and wait until we declare it stable. As you said, the same we did for `setTimeout` et al.

Fixed at https://github.com/grafana/xk6-websockets/pull/74/commits/c38822a0d53ba041d8979a93d31e0ad79ee17de1